packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -0,0 +1,292 @@
+      if (!isEmpty(hasComment)) { hasCommentIcon = hasComment; }
+      if (!isEmpty(isUnverified)) { hasUnverifiedIcon = isUnverified; }
+    }
+
+    const hasDiastolic = resultData.diastolic;
+    const noDataDiastolic = (hasDiastolic && resultData.diastolic.resultNoData === true);
+    if (hasDiastolic && !noDataDiastolic) {
+      const {
+        result = null,
+        status = null,
+        conceptDisplay = null,
+        datetimeDisplay = null,
+        isModified = null,
+        hasComment = null,
+        isUnverified = null,
+      } = hasDiastolic;
+
+      if (!isEmpty(result.unit)) { compareUnits.diastolic = result.unit.trim().toLowerCase(); }
+      if (!isEmpty(status)) { compareStatusIsInError.diastolic = checkIsStatusInError(hasDiastolic); }
+      if (!isEmpty(conceptDisplay)) { compareConceptDisplays.diastolic = conceptDisplay.trim().toLowerCase(); }
+      if (!isEmpty(datetimeDisplay)) { compareDatetimeDisplays.diastolic = datetimeDisplay.trim().toLowerCase(); }
+      if (!isEmpty(isModified)) { hasModifiedIcon = isModified; }
+      if (!isEmpty(hasComment)) { hasCommentIcon = hasComment; }
+      if (!isEmpty(isUnverified)) { hasUnverifiedIcon = isUnverified; }
+    }
+
+    if (hasSystolic || hasDiastolic) {
+      if ((compareUnits.systolic === compareUnits.diastolic) && !compareStatusIsInError.systolic) {
+        const systolicDisplay = (
+          <Observation
+            key={`Observation-Systolic-${resultData.systolic.eventId}`}
+            eventId={resultData.systolic.eventId}
+            result={resultData.systolic.result}
+            interpretation={!compareStatusIsInError.systolic ? resultData.systolic.interpretation : null}
+            isUnverified={resultData.systolic.isUnverified}
+            hideUnit
+          />
+        );
+        decoratedResultDisplay.push(systolicDisplay);
+      } else {
+        if (!hasSystolic) decoratedResultDisplay.push(<ResultError key={`Error-Systolic-${resultData.id}`} />);
+        if (noDataSystolic) decoratedResultDisplay.push(<NoData key={`NoData-Systolic-${resultData.id}`} />);
+        else if (hasSystolic) {
+          const systolicDisplay = (
+            <ConditionalWrapper
+              key={`del-Systolic-${hasSystolic.eventId}`}
+              condition={compareStatusIsInError.systolic}
+              wrapper={children => <del>{children}</del>}
+            >
+              <Observation
+                key={`Observation-Systolic-${resultData.systolic.eventId}`}
+                eventId={resultData.systolic.eventId}
+                result={resultData.systolic.result}
+                interpretation={!compareStatusIsInError.systolic ? resultData.systolic.interpretation : null}
+                isUnverified={resultData.systolic.isUnverified}
+                hideUnit={hideUnit}
+              />
+            </ConditionalWrapper>
+          );
+          decoratedResultDisplay.push(systolicDisplay);
+        }
+      }
```

Comment:
The no data & error view should be handled by the Observation component. We should be able to slim this down to the following:
```suggestion
  const unitsMatch = compareUnits.systolic === compareUnits.diastolic;
  const systolicDisplay = (
    <ConditionalWrapper
      key={`del-Systolic-${resultData.systolic.eventId}`}
      condition={compareStatusIsInError.systolic}
      wrapper={children => <del>{children}</del>}
    >
      <Observation
        key={`Observation-Systolic-${resultData.systolic.eventId}`}
        eventId={resultData.systolic.eventId}
        result={resultData.systolic.result}
        // IMO the Observation should be checking the status and handing this interpretation logic. It's repeated a few times
        interpretation={!compareStatusIsInError.systolic ? resultData.systolic.interpretation : null}
        isUnverified={resultData.systolic.isUnverified}
        hideUnit={(unitsMatch && !compareStatusIsInError.systolic) || hideUnit}
      />
    </ConditionalWrapper>
  );

  decoratedResultDisplay.push(systolicDisplay);
```